presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -283,7 +274,6 @@ public PlanOptimizers(Metadata metadata, SqlParser sqlParser, FeaturesConfig fea
                 new MetadataQueryOptimizer(metadata),
                 new IterativeOptimizer(
                         stats,
-                        ImmutableList.of(new com.facebook.presto.sql.planner.optimizations.EliminateCrossJoins()), // This can pull up Filter and Project nodes from between Joins, so we need to push them down again
```

Comment:
Not sure this comment was important, but i kind of explained why we have `PredicatePushDown` as the next optimizer. I think the comment still applies to the iterative rule the same way it applied to `EliminateCrossJoins` optimizer